__tests__/OmnichannelChatSDK.spec.ts

```diff
@@ -3131,6 +3130,99 @@ describe('Omnichannel Chat SDK, Sequential', () => {
             expect(chatSDK.IC3Client).toBe(undefined);
         });
 
+        it('ChatSDK.endChat() should end conversation without calling closeSession', async () => {
```

Comment:
`ChatSDK.endChat() with optional params \'isSessionEnded\' set as \'true\' should not call sessionClose`